arm/compute/containerservice/client.go

```diff
@@ -1,44 +0,0 @@
-// Package compute implements the Azure ARM Compute service API version
-// 2016-03-30.
```

Comment:
It depends on when core team updates CompositeSwagger modeler. ACS will be added after that.